### DISCUSSION TITLE
malware ?

### DISCUSSION THREAD

**Discussion by yoyoyman** [sumatrapdfreader/sumatrapdf#2976] [Q&A]
Hello. I guess this is a false positive, but could you explain this report on the setup SumatraPDF-3.4.6-64-install.exe downloaded from the official website 

https://www.filescan.io/reports/aa79391c7db478fbb969875da39ce09e3e8124b869acc3178f5b6a3b4e10d5ce/e888dd07-6a17-44da-b81d-a6447ee150a5/overview

thanks

**Comment by GitHubRulesOK** [comment]
Is Microsoft a virus https://www.virustotal.com/gui/ip-address/13.107.4.52/relations

Anyway see #2974
